src/app/(post-login)/submission/layout.tsx

```diff
@@ -19,33 +19,48 @@
 'use client';
 
 import { css, useTheme } from '@/lib/emotion';
-import { ReactNode } from 'react';
+import { ReactNode, useState } from 'react';
 import SideMenu from './components/Sidemenu';
 import TitleBar from './components/TitleBar';
 
-export default function SubmissionLayout({ children }: { children: ReactNode }) {
+export default function AppLayout({ children }: { children: ReactNode }) {
 	const theme = useTheme();
+	const programData: never[] = [];
+	const [isSidebarActive, setSidebarActive] = useState<boolean>(true);
 
 	return (
 		<div
 			css={css`
 				display: grid;
-				grid-template-columns: 248px 1fr;
+				grid-template-columns: ${isSidebarActive ? '248px' : '40px'} 1fr;
+				transition: 300ms;
+				background: ${theme.colors.grey_4};
 			`}
 		>
-			<SideMenu />
+			<div
+				css={css`
+					height: calc(100vh - 58px);
```

Comment:
what's this `58px` from? going to guess the navbar.
either way, both for maintainability and declarativeness, this should be a "shared" variable with whatever component is that tall.

Reply:
https://github.com/icgc-argo/rdpc-ui/pull/48/commits/48167ff884cc65567212b1c8e5e2a72ba48f3969
have it as a simple exported number for maximum portability (css, regular js func) vs '58px' or a `css` fragment